label_studio/io_storages/base_models.py

```diff
@@ -431,7 +438,8 @@ def _scan_and_create_links(self, link_class):
 
             logger.debug(f'{self}: found new key {key}')
             try:
-                tasks_data = self.get_data(key)
+                # list of (task data + ImportStorageLink details)
+                links_params = self.get_data(key)
```

Comment:
with the change to `StorageObject`, this can be `storage_objects`, which I think will be a lot more readable than `links_params` vs `link_params` (although, valiant effort at making the plurals situation readable enough!)